### ISSUE TITLE
Wordpress-Azure File Share: Images are not visible (alt texts are only visible)

### ISSUE THREAD

**Issue by the1bit** [Azure/AKS#2614]
**What happened**: I created Wordpress on AKS which uses Azure File Share as Persistent Volume. I followed during the storage creation the official article: [https://docs.microsoft.com/en-us/azure/aks/azure-files-volume](https://docs.microsoft.com/en-us/azure/aks/azure-files-volume). I can install everything fine (plugins, themes, etc). When I upload any image, it appears on Azure Storage on the right way but it has not preview. Unfortunately, images are not visible neither in Media Store nor in any post. I mean, no preview of the pictures and users cannot see them.

**What you expected to happen**: When I upload any image, the image is viible in Media Store then the  post  where I attach it.

**How to reproduce it (as minimally and precisely as possible)**: Upload image to media store. After uploading, image is not visible

**Anything else we need to know?**: If I use managed disk as Persistent Volume, all images are visible. Unfortunately, If I use Managed Disk I cannot manage wordpress files. (which is required)

**Environment**:
- Kubernetes version (use `kubectl version`): 1.21.2
- Size of cluster (how many worker nodes are in the cluster?): 3
- General description of workloads in the cluster (e.g. HTTP microservices, Java app, Ruby on Rails, machine learning, etc.): HTTP microservices: Wordpress, ingress-controller, Ingress
- Others:

**Comment by phealy** [comment]
@the1bit Uploading a file to the storage won't automatically make it appear in Wordpress; the database needs an entry for it. Your storage appears to be working properly from the AKS side. You'd need to use something like a WordPress plugin to import your uploaded files into the database - a quick search for "wordpress files not showing in media library" returns a number of results that appear relevant, but as this isn't an AKS related issue, I can't provide support for wordpress directly.

**Comment by phealy** [comment]
Followup - I reread and I see you say it works with disk. This suggests wordpress is not setting the appropriate permissions on uploaded files. Please check into that; it's nothing AKS can affect directly.

**Comment by the1bit** [comment]
Hi,

I am sure that is not WP related, because I have similar installation with managed disk, which works fine.

Additionally I use the following permission for files:

```
    readOnly: false
  mountOptions:
  - dir_mode=0777
  - file_mode=0777
  - uid=1000
  - gid=1000
  - mfsymlinks
  - nobrl
```

I tried with 0755. In that case wordpress cannot access to files at all.
Please, give me an exact suggestion.

Thanks

**Comment by andyzhangx** [comment]
try 0744 as suggested by https://wpengine.com.au/resources/troubleshooting-wordpress-media-library/

**Comment by the1bit** [comment]
I promise, I'll reconfigure with 0744 if it will work. Neverhteless at the moment it does not work.

I am waiting for you

**Comment by the1bit** [comment]
Please don't close. I am waiting for your support. Thanks
